tests/elwood_unit_test.py

```diff
@@ -37,32 +47,34 @@ def tearDown(self):
         gc.collect()
         # Delete any output parquet files.
         try:
-            os.remove(f"outputs{sep}unittests.parquet.gzip")
-        except:
+            os.remove(output_path("unittests.parquet.gzip"))
+        except FileNotFoundError as e:
```

Comment:
Ah, I had added this to handle the expected exception (file not found) vs a catch all exception. Not greatly relevant, since this is under unit tests, but not a bad idea regardless.